docs-www/package.json

```diff
@@ -14,30 +14,30 @@
     "lint:autofix": "eslint --ext .jsx,.js . --fix"
   },
   "dependencies": {
-    "@brainhubeu/gatsby-docs-kit": "3.0.8",
-    "@brainhubeu/react-file-input": "^0.8.2",
-    "gatsby": "2.20.29",
-    "gatsby-link": "2.3.0",
+    "@brainhubeu/gatsby-docs-kit": "3.2.1",
+    "@brainhubeu/react-file-input": "^0.9.1",
+    "gatsby": "2.22.15",
+    "gatsby-link": "2.4.3",
```

Comment:
I'm not sure about these two updates. Last time I tried to update them live-render feature was broken in the demo. Did you check whether it works?